### PR TITLE
feat: add entrypoint for running required steps

### PR DESCRIPTION
**Pull request by lucasvuotto**
While there, adjust DEPLOYMENT.md accordingly. Also add an environment
variable in Dockerfile to allow specifying specifics targets for
`precache` subcommand.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,8 +26,9 @@ RUN sed -i 's#"./src/cli"#"./lib/cli"#g' package.json
 
 EXPOSE 3030
 
-ENTRYPOINT [ "./bin/run" ]
-CMD [ "start" ]
+ENV RIFM_PRECACHE_TARGETS rns rates
```
Comment (AuHau) [inline Dockerfile:29]: Hmmm I am pretty sure that `ENV` is only for build-time and not run-time. So I would suggest removing this. Btw. there is also an option "all" which will precache all the services, which I would make a default if the `RIFM_PRECACHE_TARGETS` is not set.

**Comment by AuHau** [comment]
I will tackle the linting problem.

**Comment by AuHau** [comment]
@lucasvuotto I have fixed the linting in the `master` so please rebase and it should not bother you anymore ;-)

File: Dockerfile (hunk 1 above)
Comment (lucasvuotto) [reply Dockerfile:29]: From https://docs.docker.com/engine/reference/builder/#env

> The environment variables set using `ENV` will persist when a container is run from the resulting image. You can view the values using `docker inspect`, and change them using `docker run --env <key>=<value>`.

Didn't know about `all`. Will use it instead.

**Comment by lucasvuotto** [comment]
Thanks for that! Rebased.

**Pull request merged**